Jenkinsfile

```diff
@@ -124,6 +124,7 @@ pipeline {
         VERSION = "${env.GIT_BRANCH}-${env.GIT_SHORT_COMMIT}"
         CLUSTER_NAME = sh(script: "echo jenkins-pxc-${GIT_SHORT_COMMIT} | tr '[:upper:]' '[:lower:]'", , returnStdout: true).trim()
         AUTHOR_NAME  = sh(script: "echo ${CHANGE_AUTHOR_EMAIL} | awk -F'@' '{print \$1}'", , returnStdout: true).trim()
+        IMAGE_BACKUP = "perconalab/percona-xtradb-cluster-operator:k8spxc-582-backup"
```

Comment:
should we stick to the current branch's tag?

Reply:
we will remove it before merge, because backup image contains code from this branch

Reply:
got it